test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeGenerator.java

```diff
@@ -626,6 +759,49 @@ private static void generateGetterGeneric(ClassWriter cw, String[] nameAndSigVal
 		mv.visitEnd();
 	}
 
+	private static void generateStaticGetterGeneric(ClassWriter cw, String[] nameAndSigValue, String className) {
+		boolean doubleDetected = false;
+		MethodVisitor mv = cw.visitMethod(ACC_PUBLIC | ACC_STATIC, "getStaticGeneric" + nameAndSigValue[0], "(Ljava/lang/Class;)Ljava/lang/Object;", null, null);
```

Comment:
the `Ljava/lang/Class;` is not needed

Reply:
same for generic setter

Reply:
when I remove these, I'm getting nullpointerexceptions at the initalizeStaticFields stage